src/components/StyleGuide/Colors.stories.mdx

```diff
@@ -91,33 +394,59 @@ export const getColorCards = (cat) => {
 
 # NYPL Colors
 
-| Table of Contents                        |
-| ---------------------------------------- |
-| 1. [Brand Colors](#brand-colors)         |
-| 2. [Section Colors](#section-colors)     |
-| 3. [Utility Colors](#utility-colors)     |
-| 3. [Grayscale Colors](#grayscale-colors) |
-| 4. [Figma Reference](#figma-reference)   |
+| Table of Contents                              |
+| ---------------------------------------------- |
+| 1. [Overview](#overview)                       |
+| 2. [Brand Fills](brand-fills)                  |
+| 3. [NYPL Brand Colors](#nypl-brand-colors)     |
+| 4. [NYPL Section Colors](#nypl-section-colors) |
+| 5. [UI Fills](ui-fills)                        |
+| 6. [Utility Colors](#utility-colors)           |
+| 7. [Grayscale Colors](#grayscale-colors)       |
+| 8. [Figma Reference](#figma-reference)         |
+
+## Overview
+
+The Reservoir Design System (DS) the provides the NYPL branded and approved
+color palette. The NYPL color palette is used to style the DS components and
+is also available for consuming applications in the form of javascript and
+CSS design tokens. Please refer to the [Design Tokens](http://localhost:6006/?path=/story/style-guide-design-tokens--page)
```

Comment:
I'm surprised the link works on Vercel. You can update it to `[Design Tokens](/?path=/story/style-guide-design-tokens--page)`